console/package.json

```diff
@@ -101,7 +103,9 @@
     "redux-thunk": "^2.2.0",
     "sanitize-filename": "^1.6.1",
     "semver": "5.5.1",
+    "source-map-loader": "^0.2.4",
     "subscriptions-transport-ws": "^0.9.12",
+    "typescript": "^3.7.5",
```

Comment:
We'll need `tslint` as well? Or `eslint` does the job?

Reply:
Eslint does the job.